models/StreetParkingRules.js

```diff
@@ -16,7 +16,17 @@ export default class StreetParkingRules {
   }
 
 
-  switchSidesAt(dateTime) {
+  switchSidesAt(dateTime, day) {
+    if(day === 1){ // There are different rules on Sunday
```

Comment:
Is there a way we can put Sunday exception logic somehow in the rules object and then test it here? Ideally, we'd want different rules objects (line 7) for each city we support... and we wouldn't want this function to change for every city (but different cities prob. have different days whereby they get around the odd # days in a week)